overlord/configstate/configcore/corecfg.go

```diff
@@ -54,12 +54,7 @@ func coreCfg(tr Conf, key string) (result string, err error) {
 
 // supportedConfigurations will be filled in by the files (like proxy.go)
 // that handle this configuration.
```

Comment:
I wonder if we could expand the comment a bit, like: `supportedConfigurations will be filled in via init() in the files (like proxy.go) that handle configuration` - iirc I wrote this but I did not do so very well :/ I think mentioning init() makes it slightly clearer how the mechanism works. Also nice to make it empty here because it was confusing before (it looked from a first glance like it would only validate experimental settings).

Reply:
How about something like this:
```
supportedConfigurations contains a set of handled configuration keys.
The actual values are populated by `init()` functions in each module.
```